js/huobi.js

```diff
@@ -144,6 +148,10 @@ module.exports = class huobi extends Exchange {
                     'private': 'https://{hostname}',
                     'v2Public': 'https://{hostname}',
                     'v2Private': 'https://{hostname}',
+                    'statusSpot': 'https://{hostname}',
```

Comment:
I'd suggest grouping these status urls in the following manner:

```JavaScript
'hostnames': {
    'status': {
         'spot': '...',
         'future': {
             'inverse': '...', (they have inverse futures and linear futures)
             'linear': '...',
         },
         'swap': {
             'inverse': '...',
             'linear': '...',
         },
    },
}
```
And then add the handling to the `sign()` implementation...

The confusing bit here is that they erroneously call USDT-Margined contracts just _"swaps"_ but also serve futures-info from those swap endpoints as well (see how other unified methods of huobi deal with that):

<img width="1580" alt="Screenshot 2022-05-10 at 02 07 57" src="https://user-images.githubusercontent.com/1294454/167512997-8c4781c4-27cb-4552-b153-45e9e075e6f5.png">